### DISCUSSION TITLE
Linear profile even with fine mesh

### DISCUSSION THREAD

**Discussion by JaeHyun-Nam** [shannon-lab/zapdos#102] [Q&A Plasma Chemistry / CRANE] [answered]
Hello all,

I made fine mesh using Gmsh like below with special attention to gas-liquid interface. The minimum size of is ~5x10^-10m.
![image](https://user-images.githubusercontent.com/54873369/143325208-b5c09d44-52e4-4e30-8270-fb8844490412.png)

However, below results have discontinuous profiles near the interface and seem to have very coarse mesh.

![image](https://user-images.githubusercontent.com/54873369/143325325-ba1f41ef-19e3-42ff-9224-4b1e84ac54f9.png)

(but in gas phase, profile is very continuous and smooth)

How can I explain this? 

P.S. I used Paraview that opened exodus format file. Is there any possibility that this format file contains nodes-skipped values?

Thank you!

**Comment by csdechant** [comment, marked as answer]
Do you have the `.geo` file for the mesh you made in a Git branch? It is a little hard to tell which domain is the gas and which is the liquid from the picture. My first guess is that there was a refinement issue and only the gas domain got refined. As for Paraview, there have been know issues with line interpolation between nodes (usually with 2-D simulations) , but I haven't heard about Paraview skipping nodes.

**Reply by JaeHyun-Nam** [reply]
Hello, this is .geo file. 

[liquidNew_modified.txt](https://github.com/shannon-lab/zapdos/files/7610739/liquidNew_modified.txt)

This is file is from "liquidNew.geo" I made some modifications here to refine liquid domain.

**Reply by csdechant** [reply]
I ran the modified mesh with `mean_en.i` in the` /tests/1d_dc/` directory. The results for the electrons in the water are shown in the plot for both mesh versions. The mesh refinement seems to be good. Are you also running a modified version of `mean_en.i`?
![Mesh-OriginalVsModified](https://user-images.githubusercontent.com/37221357/143658301-c3e325bb-35e9-402d-b69d-6ea18582666b.png)

**Reply by Jae-Nam** [reply]
@csdechant Thank you. I also just ran refined mesh with "mean_en.i" and result seems to be good.
But with my modified input file (here I added some reactions using crane action and scaled up liquid domain 1e-7=>1e-5 m), there are still discontinuous profiles. In this case, can the problem be convergence related?
![image](https://user-images.githubusercontent.com/95106125/143659426-fff6651b-11db-4ab2-8892-f4dc5658c988.png)

**Reply by Jae-Nam** [reply]
I figured out. This was just a 'resolution' problem of Paraview. I changed to higher resolution settings and the profile became more continuous.
